server/app/constants.py

```diff
@@ -4,3 +4,4 @@
 STUDENT_ROLE = 'student'
 STAFF_ROLE = 'staff'
 VALID_ROLES = [STUDENT_ROLE, STAFF_ROLE]
+GRADES_BUCKET = 'ok_grades_bucket'
```

Comment:
This eventually won't be a constant but probably a property of a class. This is fine for now.